### DISCUSSION TITLE
Not able to analyze data with more than 10 rows!

### DISCUSSION THREAD

**Discussion by aavhd** [vaexio/vaex#1239] [Q&A]
Hi,
I am having an unusual issue with using Vaex.
When I run the example code in the docstring, it works just fine. However, I cannot work with any data type (Pandas DataFrame, CSV or even Numpy Array) with more than 10 rows!
This code works just fine:
![1](https://user-images.githubusercontent.com/80156904/110206022-a4c9a180-7e90-11eb-92ca-4ef61bc66121.jpg)
But this does not output anything at all:
![2](https://user-images.githubusercontent.com/80156904/110206029-b27f2700-7e90-11eb-82db-fdf190d08a55.jpg)
I would appreciate if you could help me solve this issue...
Amirali

**Comment by JovanVeljanoski** [comment]
I don't understand.. what is the error message?

I don't see anything from in your example. What do you want it to output? Can you try and print out the dataframe?

**Reply by aavhd** [reply]
The first code's output when I print() it, is the complete dataframe with less than 0.0s of computation.
But when I run the second one, it seems like the computation never stops. I have no output.

**Reply by JovanVeljanoski** [reply]
I can not reproduce this issue..  can you give more context, like how did you install vaex, the OS, your `vaex.__version__`, are you using an IDE (e.g. can you try running your code in a simple ipython terminal).

**Reply by aavhd** [reply]
INSTALLATION
pip install vaex

OS
Windows 10 64-bit

VAEX VERSION
{'vaex': '3.0.0', 'vaex-core': '2.0.3', 'vaex-viz': '0.4.0', 'vaex-hdf5': '0.6.0', 'vaex-server': '0.3.1', 'vaex-astro': '0.7.0', 'vaex-jupyter': '0.5.2', 'vaex-ml': '0.9.0', 'vaex-arrow': '0.5.1'}

I am currently using PyCharm but non of the other IDEs work either. VS Code. Jupyter N or even  a simple ipython terminal.

**Reply by JovanVeljanoski** [reply]
Can you try installing it fresh in a new virtual environment? 

Also what happens if you try something like:

```
df = vaex.example()
print(df)
```

**Reply by aavhd** [reply]
This is the error I get!
![image](https://user-images.githubusercontent.com/80156904/110213078-be301500-7eb3-11eb-86db-cf443fa6e162.png)

**Reply by aavhd** [reply]
I tried and it does not still work in a new virtual environment!

**Reply by JovanVeljanoski** [reply]
Ok so this tells me that it is most likely some installation issue, or pycharm is not picking up your environment. 

My advice would be to (at the beginning) start running all your tests in a ipython environment, just to eliminate that possibility. Or a jupyter notebook if that is possible. 

Maybe create a separate conda or pipenv environment and try there. It might be some dependency conflict that went unnoticed. 

Can you also print out `vaex.__file__` and see if it is pointing at the right place?

**Reply by aavhd** [reply]
The result of printing that out is the following:
C:\Users\asuS\AppData\Local\Programs\Python\Python38\lib\site-packages\vaex\__init__.py
I did create a new pipenv and it still is not working!

**Reply by aavhd** [reply]
Okay it actually worked! By creating a new conda environment I can now work with more than 10 rows. However, I still get the error with `vaex.example()`. Nevertheless, I genuinely appreciate your help. Now I can switch to conda env. Again, I really thank you for your time.

**Reply by JovanVeljanoski** [reply]
Actually.. we used to have issues with windows and python3.8. This might have been resolved, i am not sure. 

If possible to change to 3.7, that might solve all your problems actually (via pip or miniconda too).
